packages/compiler/src/server/serverlib.ts

```diff
@@ -339,6 +342,45 @@ export function createServer(host: ServerHost): Server {
     }
   }
 
+  async function updateImportsOnFileMovedOrRenamed(param: {
+    oldFilePath: string;
+    newFilePath: string;
+    openedFilePath: string;
+  }): Promise<string[]> {
+    const result: string[] = [];
+
+    const resultCompile = await compileService.compile({
+      uri: fileService.getURL(normalizePath(param.openedFilePath)),
+    });
+    if (resultCompile === undefined) {
+      return [];
+    }
+
+    const { program } = resultCompile;
+    const oldFileName = getBaseFileName(param.oldFilePath);
+    for (const diagnostic of program.diagnostics) {
+      if (diagnostic.code === "import-not-found") {
+        const target = diagnostic.target as Node;
+        if (
+          target.parent &&
+          target.kind === SyntaxKind.ImportStatement &&
+          target.path.value.includes(oldFileName)
```

Comment:
i think you need to check the last segment of the path, just include may return true for path unexpectedly. i.e. oldFileName is "node.ts", target.path is "../xxx-node.ts".